examples/table/table/app.py

```diff
@@ -65,14 +71,17 @@ def startup(self):
         self.table1 = toga.Table(
             headings=headings,
             data=bee_movies[:4],
-            style=Pack(flex=1),
-            on_select=self.on_select_handler
+            style=Pack(flex=1, padding_right=5, height=350),
```

Comment:
Setting a fixed height like this isn't a good idea; it's better to leave off the height, and make the labelbox that contains the footer labels "flex=0". That way, the tables will expand to fill the available space.